frontend/src/components/ThemedButtons/DefenceConfigurationRadioButton.tsx

```diff
@@ -0,0 +1,29 @@
+import './DefenceConfigurationRadioButton.css';
+
+function DefenceConfigurationRadioButton({
+	id,
+	name,
+	checked,
+	onChange,
+}: {
+	id: string;
+	name: string;
+	checked: boolean;
+	onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+}) {
+	return (
+		<div className="defence-radio-button" key={id}>
+			<input
+				type="radio"
+				id={id}
+				name={name}
+				value={id}
+				checked={checked}
+				onChange={onChange}
+			/>
+			<label htmlFor={id}>{name}</label>
```

Comment:
Where possible, prefer enclosing inputs inside labels, rather than using ids. According to MDN, the label text may go after the input:

```
<label>
  <input ... />
  {name}
</label>
```